tests/ui/doc/issue_1832.rs

```diff
@@ -0,0 +1,9 @@
+/// Ok: <http://www.unicode.org/reports/tr9/#Reordering_Resolved_Levels>
```

Comment:
Is there a reason why you extracted these two tests?

Reply:
Those two tests fail with rustfix because they have a suggestion that isn't machine applicable, and it's compiled with `-Dwarnings`.

https://github.com/rust-lang/rust-clippy/blob/c68af43b3dffc90b4653063c5012f039cf209784/tests/compile-test.rs#L131-L132

Reply:
That explains why some lint test files are split up into a fixable and not fixable test file. Today I learned. Would you mind renaming the `doc.rs` file to `doc-fixable.rs`? You can also do that in an additional commit ^^ then everything is really ready :upside_down_face: